web/libs/core/src/lib/hooks/useCurrentUser.ts

```diff
@@ -0,0 +1,34 @@
+import type { APIUser } from "@humansignal/core/types/user";
+import { API } from "apps/labelstudio/src/providers/ApiProvider";
+import { useAtomValue } from "jotai";
+import { atomWithQuery, queryClientAtom } from "jotai-tanstack-query";
+import { useCallback } from "react";
+
+const currentUserAtom = atomWithQuery(() => ({
+  queryKey: ["current-user"],
+  async queryFn() {
+    return await API.invoke<APIUser>("me");
+  },
+}));
+
+export function useCurrentUser() {
```

Comment:
we already have a [hook with this name](https://github.com/HumanSignal/label-studio/blob/develop/web/apps/labelstudio/src/providers/CurrentUser.jsx#L26), this would be very confusing to use and can create inconsistent states

Reply:
Explained in the description.